### ISSUE TITLE
Battery percentage displays wrong column

### ISSUE THREAD

**Issue by paulvictor** [tmux-plugins/tmux-battery#20]
The  output of ```pmset -g batt``` is

```Now drawing from 'AC Power'
-InternalBattery-0 (id=3801187)	59%; charging; 1:39 remaining present: true```.

Piping it to ``` awk 'NR==2 { gsub(/;/,""); print $2 }'``` prints (id=3801187). Can this be changed to $3, i noticed this on the new preview of OSX, so I doubt if you'll be able to reproduce it, just a heads up!!

**Comment by bruno-** [comment]
Hi,
thanks for the heads up.

I'm not sure how to handle this elegantly as we still wanna keep backwards compatibility, at least for some time.

**Comment by sfarzy** [comment]
May be it was due to the preview version. Stable version doesnt' have the id column.
```awk
Now drawing from 'Battery Power'
 -InternalBattery-0 	47%; discharging; 2:14 remaining present: true
```

I had to modify the script a little bit to display battery percentage on my machine. I don't know whether anyone else has similar issue.

```awk
pmset -g batt | awk 'NR==2 { gsub(/;/,"",$2); print $2 }'
```
Thought I would share.

**Comment by martinbeentjes** [comment]
The output of the `pmset -g batt` command has changed in macOS Sierra. I fixed this with #35.
